src/memalloc.jl

```diff
@@ -55,8 +55,8 @@ isfuncexpr(ex::Expr) =
     ex.head == :function || (ex.head == :(=) && typeof(ex.args[1]) == Expr && ex.args[1].head == :call)
 isfuncexpr(arg) = false
 
-# Support Unix command line usage like `julia Coverage.jl $(find ~/.julia/v0.3 -name "*.jl.mem")`
-if abspath(PROGRAM_FILE) == @__FILE__
+# Support Unix command line usage like `julia Coverage.jl $(find ~/.julia/v0.6 -name "*.jl.mem")`
+if abspath(PROGRAM_FILE) == Pkg.dir(joinpath("Coverage", "src", "Coverage.jl"))
```

Comment:
don't use Pkg.dir. if `PROGRAM_FILE` doesn't match `@__FILE__` but something adjacent to it, use `joinpath(dirname(@__FILE__), "Coverage.jl")`